### PR TITLE
Migrate buttons

### PR DESCRIPTION
**Pull request by kul3r4**
@sebastianbenz PTAL
Deployed on [staging](https://amp-by-example-staging.appspot.com/)

### CODE REVIEW

File: api/iframe/amp-url-api.html [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "http://bit.ly/28lya4p"
   ]
 }</textarea>
-    <button id="run" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent" onclick="sendAmpUrlRequest();">
```
Comment (sebastianbenz) [inline api/iframe/amp-url-api.html]: You need to import ampstart into this file then

File: api/iframe/amp-url-converter.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
         <input id="input" type="text">
       </div>
       <div class="fixed">
-        <button id="execute">Convert</button>
```
Comment (sebastianbenz) [inline api/iframe/amp-url-converter.html]: same as above

File: src/20_Components/amp-lightbox.html [hunk 3]
```diff
@@ -34,7 +34,7 @@
     The lighbox is shown when the user taps or clicks on an element with `on` attribute that targets
     the id of an `amp-lightbox` element.
   -->
-  <button on="tap:my-lightbox" role="button" tabindex="0">
+  <button class="ampstart-btn ampstart-btn-secondary caps" on="tap:my-lightbox" role="button" tabindex="0">
```
Comment (sebastianbenz) [inline src/20_Components/amp-lightbox.html]: I'd suggest using ampstart-btn here instead.

File: src/20_Components/amp-list.html [hunk 4]
```diff
@@ -69,7 +69,7 @@
   -->
   <amp-list width="auto" height="10" layout="fixed-height" src="<%host%>/json/examples.json" template="amp-template-id">
     <div>
-  <button overflow role="button" aria-label="Show more" class="list-overflow">
+  <button overflow role="button" aria-label="Show more" class="list-overflow ampstart-btn ampstart-btn-secondary caps">
```
Comment (sebastianbenz) [inline src/20_Components/amp-list.html]: same as above

File: src/30_Advanced/amp-user-notification_with_Server_Endpoint.html [hunk 5]
```diff
@@ -57,7 +57,7 @@
       "https://rocky-sierra-1919.herokuapp.com/amp-user-notification/api/echo/post">
       This is an amp-user-notification. It uses a backend service
       to verify if the notification has to be shown.
-     <button on="tap:amp-user-notification1.dismiss">I accept</button>
+     <button on="tap:amp-user-notification1.dismiss" class="ampstart-btn ampstart-btn-secondary caps ml1">I accept</button>
```
Comment (sebastianbenz) [inline src/30_Advanced/amp-user-notification_with_Server_Endpoint.html]: remove ampstart-btn-secondary

File: templates/css/styles.css [hunk 6]
```diff
@@ -477,19 +439,7 @@ a.card:active {
...
-  -webkit-flex-direction: row;
-  -ms-flex-direction: row;
-  flex-direction: row;
-  padding: 16px;
-  -webkit-flex-grow: 1;
-  -ms-flex-positive: 1;
-  flex-grow: 1;
-  margin: 0;
-  color: rgb(97, 97, 97);
+  flex-direction: column;
```
Comment (sebastianbenz) [inline templates/css/styles.css]: why is this still here?

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/20_Components/amp-access.html [hunk 7]
```diff
@@ -78,7 +78,7 @@
   `amp-access-hide` will default the component to be hidden and enabled later
   if the `amp-access` expression is evaluated to `TRUE` -->
   <div amp-access="NOT subscriber" amp-access-hide role="button" tabindex="0">
-    <a on="tap:amp-access.login">Login</a>
+    <a class="ampstart-btn caps" on="tap:amp-access.login">Login</a>
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: Does it make sense to convert the enclosing div into a button?

**Pull request merged**